backend/cmm_helpers.ml

```diff
@@ -1246,6 +1246,41 @@ let get_field_computed imm_or_ptr mutability ~block ~index dbg =
   Cop
     (Cload { memory_chunk; mutability; is_atomic = false }, [field_address], dbg)
 
+(* Getters for unboxed int fields *)
+
+let get_field_unboxed_int32 mutability ~block ~index dbg =
+  (* CR mixed blocks: Could a backend person check the "signed"? *)
```

Comment:
Yeah, unboxed int32 memory operations should use signed. (The difference is that loading a 32-bit memory location into a 64-bit register sign-extends.)